### PR TITLE
Security Hub Integration - CloudFormation template example

### PR DESCRIPTION
**Pull request by jshcodes**
This update provides a CloudFormation template for an example Security Hub integration deployment.

### CODE REVIEW

File: Security-Hub/cloudformation/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
++ A __t2.micro__ AWS Linux 2 instance that pre-installs the Security Hub integration application as a bootstrap.
+    - This instance is assigned an elastic IP. This is not a requirement and only included for demo purposes.
++ A SQS queue, dead-letter queue and Lambda trigger for handling detections identified as findings.
++ A Lambda handler that consumes detections posted to the SQS queue and submits findings to AWS Security Hub based.
++ Security Hub is activated for the region deployed.
+
+> Please note: You will need to manual subscribe to CrowdStrike findings in the region after deploying this template.
+
+## Prerequisites
++ Valid CrowdStrike Falcon API credentials allowing access _READ_ to Event Streams, Hosts, and Detections.
```
Comment (crowdstrikedcs) [inline Security-Hub/cloudformation/README.md:17]: No more MSSP?

Comment (crowdstrikedcs) [inline Security-Hub/cloudformation/README.md:17]: No more Host Downloads for MSSP?

Comment (jshcodes) [reply Security-Hub/cloudformation/README.md:17]: Let me check the code. When I typed this I thought it only required READ...

**Review by crowdstrikedcs — CHANGES_REQUESTED** [review]
See Comment on Sensor Download API

File: Security-Hub/cloudformation/README.md (hunk 1 above)
Comment (jshcodes) [reply Security-Hub/cloudformation/README.md:17]: One call to GetDeviceDetails, so I think we're ok.

**Review by crowdstrikedcs — APPROVED** [review]

**Pull request merged**